diff --git a/module/zfs/zfs_vfsops.c b/module/zfs/zfs_vfsops.c
index 053f3f2..a9c33b2 100644
--- a/module/zfs/zfs_vfsops.c
+++ b/module/zfs/zfs_vfsops.c
@@ -1316,7 +1316,7 @@ zfs_domount(struct super_block *sb, void
 	if (!zsb->z_issnap)
 		zfsctl_create(zsb);
 
-	zsb->z_arc_prune = arc_add_prune_callback(zpl_prune_sb, sb);
+//     zsb->z_arc_prune = arc_add_prune_callback(zpl_prune_sb, sb);
 out:
 	if (error) {
 		dmu_objset_disown(zsb->z_os, zsb);
@@ -1355,7 +1355,7 @@ zfs_umount(struct super_block *sb)
 	zfs_sb_t *zsb = sb->s_fs_info;
 	objset_t *os;
 
-	arc_remove_prune_callback(zsb->z_arc_prune);
+//     arc_remove_prune_callback(zsb->z_arc_prune);
 	VERIFY(zfs_sb_teardown(zsb, B_TRUE) == 0);
 	os = zsb->z_os;
 	bdi_destroy(sb->s_bdi);
